tools/run_tests/xds_k8s_test_driver/framework/infrastructure/traffic_director.py

```diff
@@ -68,9 +68,12 @@ def __init__(
         resource_prefix: str,
         resource_suffix: str,
         network: str = 'default',
+        compute_api_version: Optional[str] = None,
```

Comment:
Should be fine to have a default here too.

```suggestion
        compute_api_version: str = 'v1',
```